plugin/evm/vm.go

```diff
@@ -316,7 +342,7 @@ func (vm *VM) GetActivationTime() time.Time {
 
 // Initialize implements the snowman.ChainVM interface
 func (vm *VM) Initialize(
-	_ context.Context,
+	ctx context.Context,
```

Comment:
This context is the context of the `Initialize` call - not the context of the VM. It should be assumed that `ctx` could be cancelled (which would mark `vm.backgroundCtx` as cancelled).

Reply:
Ah right. I'll replace this with a `context.TODO()`.